src/gen_dragonfruit/gd_dev_apogee_sp215.cpp

```diff
@@ -1,27 +1,41 @@
 #include "gd_dev_apogee_sp215.h"
```

Comment:
It would be good to add some top level comments pointing to the datasheet. This driver is a little more complicated since it also contains two devices in one.